docs/reference/esql/esql-limitations.asciidoc

```diff
@@ -112,7 +112,10 @@ it is necessary to use the search function, like <<esql-match>>, in a <<esql-whe
 directly after the <<esql-from>> source command, or close enough to it.
 Otherwise, the query will fail with a validation error.
 Another limitation is that any <<esql-where>> command containing a full-text search function
-cannot also use disjunctions (`OR`) unless all functions used in the OR clauses are full-text functions themselves.
+cannot use disjunctions (`OR`), unless:
+
+* All functions used in the OR clauses are full-text functions themselves, or
+* Scoring is not used
```

Comment:
scoring is not a thing yet - we have not merged a PR to get it out of snapshot.
also we need to adapt this section:

https://github.com/elastic/elasticsearch/blob/18b281ea16b5eb3ff95d0213dad52d1a08117e60/docs/reference/esql/esql-limitations.asciidoc?plain=1#L134-L148

Reply:
Ah, thanks for the catch! I've fixed this in a1d3cb341088b763c03a59d65789c94917f61784.

I'm expecting scoring will be out of snapshot before this is merged. Otherwise we can revert the doc changes.

Reply:
It was already included as available, as I was counting on it to be available 🙂 . I clarified the text a bit in 9fdd24f72a7afa30a490e88642b042e19be1b297